Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/PropertyUI/MeterProperties.tsx

```diff
@@ -150,8 +150,8 @@ const MeterProperties = (props: IProps) => {
                 <Input<OpenXDA.Types.Meter> Record={props.Meter} Field={'Alias'} Feedback={'Alias must be less than 200 characters.'} Valid={valid} Setter={(meter: OpenXDA.Types.Meter) => props.StateSetter(meter)} Disabled={hasPermissions()} />
                 </div>
                 <div className="col">
-                <Select<OpenXDA.Types.Meter> Record={props.Meter} Field={'Make'} Options={makeList.map(item => { return { Value: item.Value, Label: item.AltValue ?? item.Value } })} Setter={(meter: OpenXDA.Types.Meter) => props.StateSetter(meter)} Disabled={hasPermissions()} />
-                <Select<OpenXDA.Types.Meter> Record={props.Meter} Field={'Model'} Options={modelList.map(item => { return { Value: item.Value, Label: item.AltValue ?? item.Value } })} Setter={(meter: OpenXDA.Types.Meter) => props.StateSetter(meter)} Disabled={hasPermissions()} />
+                <Select<OpenXDA.Types.Meter> Record={props.Meter} Field={'Make'} Options={makeList.map(item => { return { Value: item.Value, Label: item.AltValue ?? item.Value, SortOrder: item.SortOrder ?? 0 } }).sort((a, b) => a.SortOrder - b.SortOrder)} Setter={(meter: OpenXDA.Types.Meter) => props.StateSetter(meter)} Disabled={hasPermissions()} />
```

Comment:
Technically this fixes the issue. However this not resolving the underlying problem that `modelList` and `makeList` are sorted incorrectly when they come in. (and potentially all other ValueLists)
Take a look at https://github.com/GridProtectionAlliance/SystemCenter/blob/1d8c707c5a78a13da0fb5775ed61f522369a445c/Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/PropertyUI/MeterProperties.tsx#L69
I would recommend trying to find the C# function that get's called serverside and ensuring the data is sorted correctly coming out of there